modules/olm-fb-catalogs-schemas.adoc

```diff
@@ -117,9 +117,10 @@ It is valid for an entry's replaces value to reference another bundle name that
 ----
 ====
 
-////
-For more information about defining upgrade edges, see the upgrade graph reference documentation.
-////
+[WARNING]
+====
+When using the `skipRange` field, the skipped Operator versions are pruned from the update graph and are therefore no longer installable by users with the `spec.startingCSV` property of `Subscription` objects. If you want to have direct (one version increment) updates to an Operator version from multiple previous versions, but also keep those previous versions available to users for installation, always use the `skipRange` field in addition to the `replaces` field pointing to the immediate previous version of the Operator version in question.
```

Comment:
```suggestion
When using the `skipRange` field, the skipped Operator versions are pruned from the update graph and are therefore no longer installable by users with the `spec.startingCSV` property of `Subscription` objects. If you want to have direct (one version increment) updates to an Operator version from multiple previous versions, and also keep those previous versions available to users for installation, always use the `skipRange` field in addition to the `replaces` field that points to the immediate previous version of the Operator version in question.
```

Reply:
I think "and" is more suitable here because you want to keep both the one version increments and keep the previous versions available. I think "but" sets the reader up for more of a contrast.  My .02, take or leave :)

I was also thinking, "that points" is clearer reading to me? Again, my .02.